submissions/OwoeyeGbenga/landing_page/css/style.css

```diff
@@ -0,0 +1,387 @@
+*, *:before, *:after {
```

Comment:
Nice work here, I think it will be better to leave a whitespace between each block in this file to make it easier to read.

Reply:
Done!!